libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Converters/ITemplateActivityConverter.cs

```diff
@@ -12,16 +15,21 @@ namespace Microsoft.Bot.Builder.Dialogs.Adaptive
     /// <summary>
     /// JsonConverter to load ITemplate&lt;Activity&gt;.
     /// </summary>
-    public class ActivityTemplateConverter : JsonConverter
+    public class ITemplateActivityConverter : InterfaceConverter<ITemplate<Activity>>
```

Comment:
Having a class that's "I" makes my head hurt. Are there alternatives? TemplateActivityConverterBase? Or similar